packaging/templates/mainnet-v1/sentry/sentry/bor/pbss_config.toml

```diff
@@ -51,20 +52,20 @@ gcmode = "archive"
 
 [txpool]
     nolocals = true
+    pricelimit = 25000000000
     accountslots = 16
-    globalslots = 131072
-    accountqueue = 64
-    globalqueue = 131072
+    globalslots = 32768
+    accountqueue = 16
+    globalqueue = 32768
```

Comment:
Why were these changes made? Did someone from the @maticnetwork/pos-v1 asked for it?

Reply:
No, reverted, copy issue with profile and test profiles